data/json/items/gun/shot.json

```
@@ -74,7 +74,7 @@
     "looks_like": "remington_870",
     "type": "GUN",
     "name": { "str": "handmade lever shotgun" },
-    "description": "A short homemade lever-action shotgun with a small internal tube magazine.  While still a primitive pipe and 2x4 design, it is a formiddable shotgun in its own right with room for improvement.",
+    "description": "A short homemade lever-action shotgun with a small internal tube magazine.  While still a primitive pipe and 2x4 design, it is a formidable shotgun in its own right with room for improvement.",
```

```suggestion
    "description": "A short, homemade lever-action shotgun with a small internal tube magazine.  While still a primitive design - made from a pipe and a 2x4 - it is a formidable shotgun in its own right with room for improvement.",
```
Not sure if "a 2x4" is quite correct according to CDDA practice - will need to check whether it's supposed to be something like "a plank".

---

#47709 I'll replace it locally